authors/apps/authentication/tests/test_social_auth.py

```diff
@@ -0,0 +1,65 @@
+import os
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase, APIClient
+
+
+
+class Test_Social_Authentication(APITestCase):
+    """Tests social authentication"""
+    url = reverse('authentication:social_login')
+
+    def setUp(self):
+        self.provider = "twitter"
+        self.data = {
+            "access_token": os.environ['TWITTER_ACCESS_TOKEN'],
+            "access_token_secret": os.environ['TWITTER_ACCESS_TOKEN_SECRET'],
+            "provider": self.provider
+        }
+
+    def test_user_successful_social_login(self):
+        """Test for successful user login"""
+        response = self.client.post(self.url, self.data, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Is there a way for you test for the response messages too? I think it would be best if we tested both the status code and the response message

Reply:
Working on it

Reply:
@ElMonstro, how far have you gone with implementing this feedback?